core/files/css/base.php

```diff
@@ -442,8 +441,8 @@ public function parse_property_placeholder( array $control, $value, array $contr
 			$value = call_user_func( $value_callback, $control );
 		}
 
-		// If the control value is empty, check for global default.
-		if ( ! $value ) {
+		// If the control value is empty, check for global default. `0` (integer, string) are falsy but are valid values.
+		if ( empty( $value ) && '0' !== $value && 0 !== $value ) {
```

Comment:
Maybe use `Utils::is_empty()`?

Reply:
Not good. It handles arrays in a way that is not compatible for the use here

Reply:
Why double checking? `&& '0' !== $value && 0 !== $value`

Reply:
Because I specifically want to handle these two cases.
Casting to an `int` in this comparison can return true for non-wanted values such as `false`, or even `'false'`.